runelite-client/src/main/java/net/runelite/client/plugins/inventoryviewer/InventoryViewerOverlay.java

```diff
@@ -61,12 +64,22 @@ private InventoryViewerOverlay(Client client, ItemManager itemManager, Inventory
 		panelComponent.setOrientation(ComponentOrientation.HORIZONTAL);
 		this.itemManager = itemManager;
 		this.client = client;
+		this.config = config;
 		this.hidden = config.hiddenDefault();
 	}
 
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
+		if (config.hiddenInventory())
+		{
+			Widget inventoryWidget = client.getWidget(WidgetInfo.INVENTORY);
+			if (inventoryWidget != null && !inventoryWidget.isHidden())
```

Comment:
This should check `client.getVar(VarClientInt.INVENTORY_TAB)` rather than getting the widget. See usages of this in the [music plugin](https://github.com/runelite/runelite/blob/088cb19752169acefef5ac0df88dcbfcd6645f39/runelite-client/src/main/java/net/runelite/client/plugins/music/MusicPlugin.java#L333-L336) and the [quest plugin](https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/questlist/QuestListPlugin.java#L211-L214)